tensorflow/lite/experimental/micro/kernels/activation_utils.h

```diff
@@ -33,9 +33,18 @@ inline float ActivationValFloat(TfLiteFusedActivation act, float a) {
       return a;
     case kTfLiteActRelu:
       return a < 0.f ? 0.f : a;
+    case kTfLiteActRelu1:
+      return a < 0.f ? 0.f : ((a > 1.f) ? 1.f : a);
+    case kTfLiteActRelu6:
+      return a < 0.f ? 0.f : ((a > 6.f) ? 6.f : a);
+    case kTfLiteActTanh:
+      return (expf(a) - expf(-a)) / (expf(a) + expf(-a));
+    case kTfLiteActSignBit:
+      return signbit(a);
```

Comment:
Note that in C++ (std::)signbit returns true if the parameter is *negative*, false if it's positive, so this is a totally useless activation function.

It's even worse in C: it returns *any* nonzero value if the value is negative. It can be 1, 255, -128, etc..